SECURITY.md

```diff
@@ -0,0 +1,49 @@
+<!--- https://www.eclipse.org/security/ --->
```

Comment:
Same as above (actually: Where is this coming from?)

Reply:
As we discussed, we will not point it here.